### ISSUE TITLE
auto-opening

### ISSUE THREAD

**Issue by Akuli** [ThePhilgrim/minesweeper#14]
When you open 0, it should open all of its neighbors

**Comment by ThePhilgrim** [comment]
I will look at this tomorrow! :D

**Comment by Akuli** [comment]
Currently it's drawing `button_image_pressed` on top of already opened squares, and that makes the numbers not visible:

![ss](https://user-images.githubusercontent.com/18505570/104808377-6a2b7d00-57ee-11eb-927d-7f3d15fdfccf.png)

I would suggest making a function `open_square()` that works like this:
```
if the square has not been opened already:   # use list of already opened squares for this
    draw button_image_pressed
    add number
    if added number was zero:
        call `open_square()` for each neighbor square
```

**Comment by ThePhilgrim** [comment]
I just realized that it doesn't display numbers.. It should only display the number if it's not 0 though.

**Comment by Akuli** [comment]
If you want to undo your last commit and start from scratch, you can run `git revert HEAD`.

**Comment by ThePhilgrim** [comment]
I also see that the numbers look like they cross the bottom square edge on your system. They don't on mine

**Comment by Akuli** [comment]
They don't look like that for me either. That screenshot is cropped to not include the entire window.

**Comment by Akuli** [comment]
I'm on ##learnpython now in case you need help. I will also be there tomorrow at about the same time.

**Comment by Akuli** [comment]
fixed in 63b678120b612ce350cd03a17dee4840e8d47b16
